### ISSUE TITLE
Interesting Test Errors

### ISSUE THREAD

**Issue by FreeBreadsticks** [learn-co-curriculum/react-rendering-lab#9]
I'm mainly posting this in case anyone else comes across the same issues I did. The first error I received preventing me from running the tests at all:
```
/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180
          throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
          ^

ReferenceError: Unknown plugin "transform-react-inline-elements" specified in "/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/.babelrc" at 3, attempted to resolve relative to "/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000"
    at /Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at compile (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-register/lib/node.js:103:45)
    at loader (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/mocha/lib/mocha.js:514:10)
    at Object.<anonymous> (/Users/erikahughes/Development/code/labs/react-rendering-lab-v-000/node_modules/mocha/bin/_mocha:480:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
/Users/erikahughes/.rvm/gems/ruby-2.3.1/gems/learn-test-2.4.2/lib/learn_test/strategies/mocha.rb:49:in `results': undefined method `[]' for nil:NilClass (NoMethodError)
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/gems/learn-test-2.4.2/lib/learn_test/runner.rb:72:in `push_results'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/gems/learn-test-2.4.2/lib/learn_test/runner.rb:19:in `run'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/gems/learn-test-2.4.2/bin/learn-test:60:in `<top (required)>'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/bin/learn-test:23:in `load'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/bin/learn-test:23:in `<main>'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/erikahughes/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'
```
Fixed using this command: `npm install --save-dev babel-plugin-transform-react-inline-elements`

Next strange error I came across was: 
```
 Error: 
      Enzyme Internal Error: Enzyme expects an adapter to be configured, but found none. To
      configure an adapter, you should call `Enzyme.configure({ adapter: new Adapter() })`
      before using any of Enzyme's top level APIs, where `Adapter` is the adapter
      corresponding to the library currently being tested. For example:

      import Adapter from 'enzyme-adapter-react-15';

      To find out more about this, see http://airbnb.io/enzyme/docs/installation/index.html
    
      at validateAdapter (node_modules/enzyme/build/validateAdapter.js:14:11)
      at getAdapter (node_modules/enzyme/build/Utils.js:71:36)
      at new ShallowWrapper (node_modules/enzyme/build/ShallowWrapper.js:117:44)
      at shallow (node_modules/enzyme/build/shallow.js:19:10)
      at Context.<anonymous> (test/index-test.js:14:21)
```
This came up for every test. Here was how we were able to solve that:
In `test/root.js` right under `require('babel-register')();
`

```
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
```

**Comment by Enoch2k2** [comment]
That was extremely helpful! :)

**Comment by lisaychuang** [comment]
Bumping this solution, as of 11/8 I still encountered the issue with tests and @FreeBreadsticks solution works!  Thank you.

**Comment by hansenjl** [comment]
I also encountered this same exact issue working with @vanitanaidu on the full stack web dev. with React track but these fixes did not work. After implementing the noted fixes above, she got the error shown in the screenshot:
![screen shot 2017-11-11 at 7 13 20 pm](https://user-images.githubusercontent.com/28542439/32696024-03e325f4-c721-11e7-908a-53a90329949f.png)

When the above fixes didn't work, we tried changing the version of enzyme but still ran into error messages.
